test/functional/apps/discover/group4/_esql_view.ts

```diff
@@ -257,7 +257,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
     });
 
-    // FLAKY: https://github.com/elastic/kibana/issues/183847
     describe.skip('inspector', () => {
```

Comment:
The flaky comment was removed, but it looks like the tests are still skipped in this PR.

Reply:
🤦‍♀️